packages/workspace/src/generators/library/library.ts

```diff
@@ -142,20 +148,26 @@ export async function libraryGenerator(tree: Tree, schema: Schema) {
   }
   addProject(tree, options);
 
-  let installTask: GeneratorCallback;
+  const tasks: GeneratorCallback[] = [];
 
   if (options.linter !== 'none') {
-    installTask = await addLint(tree, options);
+    const lintCallback = await addLint(tree, options);
+    tasks.push(lintCallback);
   }
   if (options.unitTestRunner === 'jest') {
-    installTask = (await addJest(tree, options)) || installTask;
+    const jestCallback = await addJest(tree, options);
+    tasks.push(jestCallback);
   }
 
   if (!options.skipFormat) {
     await formatFiles(tree);
   }
 
-  return installTask;
+  return async () => {
```

Comment:
Do we want to add a helper function for this?

Or more succinctly we could do:

```
return () => Promise.all(tasks.map(task => task()));
```